.dockerignore

```diff
@@ -0,0 +1,2 @@
+.github
+.node_modules
```

Comment:
shouldn't have the leading dot for node_modules

Reply:
fixed